cql-engine-fhir/pom.xml

```diff
@@ -35,6 +35,12 @@
             <artifactId>hapi-fhir-jpaserver-base</artifactId>
             <version>${hapi.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ca.uhn.hapi.fhir</groupId>
```

Comment:
Why was this dependency needed? It should be included in hapi-fhir-jpaserver-base.